backend/app/controllers/api/v1/accounts/project_developers_controller.rb

```diff
@@ -2,12 +2,12 @@ module API
   module V1
     module Accounts
       class ProjectDevelopersController < BaseController
-        skip_before_action :require_json!, only: :create
-        before_action :authenticate_user!
+        skip_before_action :require_json!, only: %i[create update]
+        before_action :require_project_developer!, except: :create
```

Comment:
I would whitelist, especially if there is only one action to whitelist.

Reply:
But that is just opinion ;]

Reply:
Or maybe, no, you are right, better to require by default here :+1: and use `except`